guillotina_amqp/state.py

```diff
@@ -372,3 +386,48 @@ def __init__(self, task_id):
     async def is_canceled(self):
         util = get_state_manager()
         return await util.is_canceled(self.task_id)
+
+
+async def update_task_status(state_manager, task_id, status, task=None, ttl=None,
```

Comment:
@vangheem I'm thinking whether we should eventually expire all data from redis... at least after a long period of time, like 24h?

Reply:
That is probably fine to set a default ttl.

Although, as long as redis is configured with an eviction policy, it should be fine to not.